src/classes/OPP_OpportunityContactRoles_TEST.cls

```diff
@@ -92,9 +94,13 @@ private class OPP_OpportunityContactRoles_TEST {
         );
         insert opp2;
 
-        result = [select OpportunityId, ContactId, Role from OpportunityContactRole where OpportunityId = :opp2.Id];
+        result = [SELECT OpportunityId, ContactId, Role FROM OpportunityContactRole WHERE OpportunityId = :opp2.Id];
         system.assertEquals(con.Id, result[0].ContactId);
-        system.assertEquals(donorRoleforTest, result[0].Role);
+        if (OPP_AutomatedSoftCreditsService.isOrganizationalAccount(acc.npe01__SYSTEMIsIndividual__c)) {
```

Comment:
since you are explicitly setting acc.npe01__SYSTEMIsIndividual__c = false at the beginning of this test method, this if should always be true, and the else clause will never get tested.  Shouldn't we have two separate tests; one for household account donations, and one for organizational donations.  they each tests could assert that the correct role is created.

Reply:
I added a test since we now have different handling for Individual and Organizational Accounts.